seumich/models.py

```diff
@@ -369,7 +370,8 @@ class StudentClassSiteAssignment(models.Model, SeumichDataMixin):
         db_column='CLASS_ASSGN_PNTS_ERND_NBR')
     included_in_grade = models.CharField(max_length=1,
                                          db_column='INCL_IN_CLASS_GRD_IND')
-    grader_comment = models.CharField(max_length=4000, null=True,
+    # Identifier name modified to be distinct from new function name
+    raw_grader_comment = models.CharField(max_length=4000, null=True,
```

Comment:
This is probably debatable, but I'd prefer to see the field name stay as closer to the database field name and instead make the new property a different, expanded name, like `clean_grader_comment` or `formatted_grader_comment`.